python/ray/tests/test_object_spilling_threshold.py

```diff
@@ -0,0 +1,53 @@
+import pytest
```

Comment:
Could you add these tests to `test_object_spilling_2.py` instead? Just to avoid cluttering the pytests.